apps/server/src/modules/oauth-adapter/service/oauth-adapter.service.ts

```diff
@@ -53,10 +53,7 @@ export class OauthAdapterService {
 		try {
 			responseToken = await lastValueFrom(observable);
 		} catch (error: unknown) {
-			if (isAxiosError(error)) {
-				throw new TokenRequestLoggableException(error);
-			}
-			throw error;
+			throw new OAuthAdapterErrorLoggableException(String(error));
```

Comment:
Why String(error) ?
If you want to send a text message, as the constructor wants, then do so